src/content/pages/account/login/index.tsx

```diff
@@ -111,12 +118,43 @@ const LoginForm = () => {
 
         // Don't continue to login logic if there are errors.
         // We cannot check the useState values, as these will not be updated until the next render.
-        if (usernameInvalid || passwordInvalid) return
+        //if (usernameInvalid || passwordInvalid) return
```

Comment:
Is this supposed to be commented out?

Reply:
Nah, was just commented out temporarily. Fixed.